target/linux/layerscape/armv8_32b/target.mk

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (C) 2016 Jiang Yutang <jiangyutang1978@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+ARCH:=arm
+BOARDNAME:=ARMv8 32-bit based boards
+CPU_TYPE:=cortex-a9
```

Comment:
@chunkeey 
Sorry for my late response. I just come back to office today from holiday.
I didn't insist A9 ... In current layerscape platforms, some are with A53 cpus and some are with A72 cpus. More details,
http://www.nxp.com/products/microcontrollers-and-processors/arm-processors/qoriq-layerscape-arm-processors:QORIQ-ARM

I don't know which CPU_FLAGS is proper for *all* of them and doesn't introduce any potential issue, but at least platforms worked fine with current A9.
You suggested to use A7 or A15, which one exactly should I use in your view?
Should I also add CPU_SUBTYPE:=neon-vfpv4 ?
I really would like to fix it with proper value.

Regarding to BOARDNAME, I think it seems proper because they are indeed ARMv8 32-bit.

@hauke @nbd168 @fsl-jyt 
Do you have any suggestion on CPU_TYPE?
Thanks a lot :)